index.js

```diff
@@ -95,7 +94,7 @@ var wcI18nV1 = function(file, encoding, callback) {
 var wcI18nV2 = function(file, encoding, callback) {
```

Comment:
I think it's a little cleaner to put these versions in separate files and then index becomes a thin wrapper of requiring the desired one.  Makes it a little easier to test.